R/kde1d_methods.R

```diff
@@ -37,6 +37,17 @@ dkde1d <- function(x, obj) {
         x <- x[[1]]
     if (!is.ordered(x))
         stopifnot(!is.factor(x))
+    check_boundary_violations(x, obj$xmin, obj$xmax)
```

Comment:
Shouldn't it rather assign density zero outside the boundary?

Reply:
you're right that's better.